scrapy/__init__.py

```diff
@@ -35,5 +35,6 @@
 from scrapy.http import Request, FormRequest
 from scrapy.selector import Selector
 from scrapy.item import Item, Field
+from scrapy.pipelines import ItemPipeline
```

Comment:
I meant the general test suite. But you're right, for consistency your way is better